.dockerignore

```diff
@@ -57,6 +57,7 @@ packages/adapter-prisma/prisma/migrations
 db.sqlite
 packages/adapter-supabase/supabase/.branches
 packages/adapter-drizzle/.drizzle
+pglite-data/
```

Comment:
```suggestion
```

Reply:
pglite generates a folder automatically, a lot of files will be committed if we don't ignore it